boa/contracts/vyper/ir_executor.py

```diff
@@ -839,10 +841,11 @@ class Assert(IRExecutor):
 
     def _compile(self, test):
         _ = VMRevert  # make flake8 happy
+        source = self.ir_node.ast_source
```

Comment:
@charles-cooper is this the right property? source_pos is gone.

Reply:
should be

Reply:
source_pos was a tuple[int,int,int,int] this seems to be str though

Reply:
it should be getpos(ast_source)

Reply:
or we could refactor, since we have the ast map now. it's set here for the code in `vyper_contract.py` to find later, so we could refactor this to `ast_node`

Reply:
As discussed elsewhere
> Charles Cooper, [13-05-2024 19:19]
> Yea I guess getpos is safest, we can update it later too